README.md

```diff
@@ -1 +1,187 @@
-# Hello-Microverse
\ No newline at end of file
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Hello Microverse README</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ ](#faq)
+- [📝 License](#license)
+
+
+
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+ **Hello Microverse** a simple assigment for Microverse to  test the use of linters and github.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Presentation</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+ The following are key features that are displayed on the portfolio
+
+- **headers**
+- **paragraphs**
+- **html** 
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon
```

Comment:
- [ ] If your project doesn't have a live demo there is no need for this section, so you can remove it. :+1: 
- [ ] In removing it make sure to make the same changes to the `Table of content` where you have `live demo` and remove it too :+1: :sparkles: